### PR TITLE
Use a short default wait for cloud deployment

### PR DESCRIPTION
**Pull request by mpolden**
@jonmv

### CODE REVIEW

File: client/go/internal/cli/cmd/deploy.go [hunk 1]
```diff
@@ -73,6 +72,12 @@ $ vespa deploy -t cloud -z perf.aws-us-east-1c`,
 					return err
 				}
 			}
+			if opts.Target.IsCloud() && waitSecs == 0 && !cmd.PersistentFlags().Changed("wait") {
+				// If --wait is not explicitly given, we always wait a few seconds in Cloud to catch fast failures, e.g.
+				// invalid application package
+				waitSecs = 2
```
Comment (jonmv) [inline client/go/internal/cli/cmd/deploy.go:78]: 2sec is fine for small packages, but probably not for larger stuff. But I guess we want people to use a larger --wait?

**Review by jonmv — DISMISSED** [review]

File: client/go/internal/cli/cmd/deploy.go (hunk 1 above)
Comment (jonmv) [reply client/go/internal/cli/cmd/deploy.go:78]: Is 10s too long? Probably ...

Comment (mpolden) [reply client/go/internal/cli/cmd/deploy.go:78]: Thanks! The suggestion was to add a short wait that catches fast errors in most cases. However, it shouldn't fail if this short wait elapses. I'll fix that.

**Comment by mpolden** [comment]
PTAL

**Comment by jonmv** [comment]
👍

**Pull request merged**